employees/templates/employees/report_list.html

```diff
@@ -13,46 +13,35 @@
 {% endblock %}
 
 {% block content %}
-{% include "employees/month_navigation_bar.html" %}
-<h1>{{ UI_text.PAGE_TITLE.value }} {{ title_date }}</h1>
+  <br/>
+  <h1>{{ UI_text.PAGE_TITLE.value }}</h1>
+  <h2>{{ title_date }}</h2>
+  {% include "employees/common/month_navigation_bar.html" %}
+  <br/><br/>
 
-<input class="hidden-print" type="button" id="opener" value="{{ UI_text.JOIN_PROJECT_BUTTON.value }}"/>
-
-<div class="modal-dialog hidden-print">
-    {% for error in errors.non_field_errors %}
-    <h4><font color="red">{{ error }}</font></h4>
-    {% endfor %}
-    <form action="{{ path }}" method="POST">
-        {% csrf_token %}
-        {{ form|crispy }}
-        <a href="https://guides.github.com/features/mastering-markdown/" target="_blank">Markdown is supported</a><br/>
-       <input type="submit" value="{{ UI_text.CREATE_REPORT_BUTTON.value }}">
-    </form>
-</div>
-
-<div class="hidden-print" id="dialog" style="display: none" title="{{ UI_text.JOIN_POPUP_HEADER.value }}" align="center">
-    <form action="{{ path }}" id="join_form" method="POST">
-        {% csrf_token %}
-        {{ project_form }}
-        <input type="submit" name="join" value="{{ UI_text.JOIN_POPUP_YES.value }}">
-    </form>
-    {% if hide_join %}
-        {{ UI_text.NO_PROJECTS_TO_JOIN.value }}
-    {% endif %}
-</div>
-
-</br>
-<div class="container">
-    <div class="table-responsive">
-        <table class="table">
-            <thead>
+  <div class="container main-white-container">
+    <br/>
```

Comment:
Also, please avoid using `<br/>` to anything else than breaking text inside the text. Instead of that use `<p>` or CSS styles to achieve the same effect.

Reply:
Done